ang/civicase/CaseOverview.js

```diff
@@ -40,7 +40,8 @@
    * @param {Object} $scope
    * @param {crmApi} Object
    */
-  function civicaseCaseOverviewController ($scope, crmApi) {
+  function civicaseCaseOverviewController ($scope, crmApi, BrowserCache) {
+    var browserCacheContainerName = 'civicase.CaseOverview.disabledCaseStatuses';
```

Comment:
This is a constant, so it should be in block letters.
`var BROWSER_CACHE_CONTAINER_NAME`.

What do you think of the following name
`var BROWSER_CACHE_IDENTIFIER`.? (minor comment)

Reply:
Also there should be a liner break between `var` and `$scope` variables.